server/server.go

```diff
@@ -487,7 +555,8 @@ func (s *Server) leadershipAcquired() error {
 	s.leaderSub = sub
 
 	atomic.StoreInt64(&(s.getRaft().leader), 1)
-	return nil
+
+	return s.createActivityStream()
```

Comment:
So if stream creation fails at this point, what happens? Does it trigger a new leader election?

Reply:
In that case the server panics. Which is not great. Other options include stepping down as a leader (see https://github.com/ably-forks/liftbridge/blob/e8f318dbeac5a10822dbb1ceefc214961e4a25f1/server/server.go#L461), and only logging the error and continuing (which doesn't seem amazing either).

Reply:
Following our discussion, I'm postponing this until we have feedback from upstream.

Reply:
Implemented leader step down in ff58a73.